standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestTablesCreateDropAlterTruncate.java

```diff
@@ -262,10 +262,6 @@ public void testCreateGetDeleteTable() throws Exception {
     createdTable.unsetId();
     Assert.assertEquals("create/get table data", table, createdTable);
 
-    // Check that the directory is created
-    Assert.assertTrue("The directory should not be created",
```

Comment:
When the table type is not VIRTUAL_VIEW, the table location will be modified by `getDnsPath`, so this judgment is meaningless.

https://github.com/apache/hive/blob/8a4f5ce7275842ff4f1cc917c7a2a48dde71bf4c/standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java#L2321-L2331

Reply:
Agreed. The root cause is this test used a wrong type `view` and then wrote a wrong assert.